index.js

```diff
@@ -12,46 +12,45 @@ var options = {
 
 module.exports = function(privateKey, remoteIP, response, cb) {
   var error = null;
-  
+
   if (!privateKey) {
     error = 'Private key is required';
-  } else if (!remoteIP) {
```

Comment:
Just removing the conditions is not enough in this case. If remoteIP is optional now, we should make it also omitable in the call itself.